helpers/systemd

```diff
@@ -25,7 +25,10 @@ ynh_add_systemd_config() {
 
     ynh_add_config --template="$template" --destination="/etc/systemd/system/$service.service"
 
-    systemctl enable $service --quiet
+    # do not enable a service disabled by the user
+    if ! yunohost service status $service | grep -q "start_on_boot: disabled"; then
+        systemctl enable $service --quiet
+    fi
```

Comment:
Hmmm I think that part should be entirely removed ... the "start_on_boot" thing is precisely the systemd's enabled/disabled flag, cf https://github.com/YunoHost/yunohost/blob/bookworm/src/service.py#L433

so this code is basically "if the service is enabled, enable it"


```suggestion
```

Reply:
So how can we do to not re-enable an intentionally disabled service on update?

Reply:
We just remove the original `systemctl enable $service` like you already did ? ;P

Reply:
hummm yes x)
why is he here in the first place?

Reply:
Because we did not really think about the use case of "the user wants the service to be disabled" i think

Reply:
ah, but now if a packages uses ynh_add_systemd_config at installation, the service will not be enabled :/

Reply:
the issue is during upgrade, not install